browser/images/lc_feedback.svg

```diff
@@ -1 +1 @@
+<svg height="24" viewBox="0 0 6.3499999 6.35" width="24" xmlns="http://www.w3.org/2000/svg"><g fill="#3a3a38" stroke-width=".264583"><path d="m3.8364583 1.5875a1.190625 1.190625 0 0 0 -1.190625 1.190625 1.190625 1.190625 0 0 0 1.190625 1.190625 1.190625 1.190625 0 0 0 1.190625-1.190625 1.190625 1.190625 0 0 0 -1.190625-1.190625zm0 .2645833a.92604166.92604166 0 0 1 .9260417.9260417.92604166.92604166 0 0 1 -.9260417.9260417.92604166.92604166 0 0 1 -.9260416-.9260417.92604166.92604166 0 0 1 .9260416-.9260417z"/><path d="m3.8364583 3.96875c-1.0884457.0006752-2.0201895.7807195-2.212268 1.8520833h4.4255696c-.1921475-1.0717475-1.1244658-1.8519083-2.2133016-1.8520833zm0 .2645833c.8408711.0000847 1.5903966.5301359 1.8706869 1.3229167h-3.7403402c.2801638-.7924234 1.0291616-1.3223942 1.8696533-1.3229167z"/><path d="m.52916667.52916667v2.91041663h.82682293l.8929687.79375.396875-.2645833-.35295-.5291667h.2485636c-.0431742-.0845935-.0779634-.1732108-.1038696-.2645833l-.6464722-.0015503.4325318.6671427-.7684286-.6655924h-.6614583v-2.38125h2.9104167v.5358846c.043972-.00425.088115-.00649.1322916-.00672.044169.0000583.088311.00213.1322917.0062v-.79994793z"/></g></svg>
```

Comment:
`viewBox="0 0 6.3499999 6.35"` could you fix this to 0 0 24 24?

Reply:
what's that for?

Reply:
and how to git ammend?